app/uk/gov/hmrc/pillar2externalteststub/models/UKTRSubscriptionRequest.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pillar2externalteststub.models
+
+import play.api.libs.json.{Json, OWrites}
+
+case class UKTRSubscriptionRequest(
+  accountingPeriodFrom: String,
+  accountingPeriodTo:   String,
+  qualifyingGroup:      Boolean,
+  obligationDTT:        Boolean,
+  obligationMTT:        Boolean,
+  liabilities:          Liabilities
```

Comment:
you are missing NilReturn here. You could look at the submission-api implementation and copy from there if it is easier

Reply:
It's okay I did it in the stubs we currently so I'll copy that

Reply:
I looked again into the spec in the ticket and there no mention of a NilReturn

Reply:
yes, I think I am getting confused with another ticket

Reply:
Okay